production/kubernetes/agent-tempo.yaml

```diff
@@ -0,0 +1,148 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: grafana-agent-traces
+---
+apiVersion: v1
+data:
+  agent.yaml: |
+    server:
+        http_listen_port: 8080
+        log_level: info
+    tempo:
+        push_config:
+            basic_auth:
+                password: ${TEMPO_PASSWORD}
+                username: ${TEMPO_USERNAME}
+            batch:
+                send_batch_size: 1000
+                timeout: 5s
+            endpoint: ${TEMPO_ENDPOINT}
+            queue:
+                retry_on_failure: false
+        receivers:
+            jaeger:
```

Comment:
how do we provide a mechanism for configuring receivers?

the jaeger agent normally starts with thrift compact/thrift binary.

Reply:
IMO the install script should be extremely opinionated, we can change the defaults though. If users want anything else they're free to manually edit the manifest, generate their own (with `make example-kubernetes`), or use the Tanka library.

I can add `thrift_binary` into the mix.